static/src/js/zustand/types.ts

```diff
@@ -876,6 +876,37 @@ export type UiSlice = {
   }
 }
 
+/** Holds project panel state */
```

Comment:
Writing 3 separate comments doesn't work either, they need to be one comment. You can do multi-line comments with `/** */` syntax, look at jsdoc comment blocks.

But you added 3 new types and didn't comment them. Adding the new types lets you better describe what they are, and ActivePanelConfiguration won't need to explain each number